packages/react-kit/src/components/form/Upload/UploadedFile.tsx

```diff
@@ -1,68 +1,116 @@
+const AttachmentContainer = styled.div<{
+  $isLeftAligned: boolean;
+  $isPdfFile?: boolean;
+}>`
   position: relative;
   display: flex;
   cursor: pointer;
   align-items: center;
-  padding: 1rem;
-  background-color: ${({ $isLeftAligned }) =>
-    $isLeftAligned ? "inherit" : colors.lightGrey};
-  ${({ $isLeftAligned }) =>
-    $isLeftAligned ? `border: 2px solid ${colors.white}` : ""};
+
   color: ${({ $isLeftAligned }) => ($isLeftAligned ? "inherit" : colors.black)};
-  margin-bottom: 0.3rem;
+
+  ${({ $isPdfFile, $isLeftAligned }) =>
+    $isPdfFile
+      ? css`
+          width: 100%;
+        `
+      : css`
+          padding: 1rem;
+          background-color: ${$isLeftAligned ? "inherit" : colors.lightGrey};
+          ${$isLeftAligned ? `border: 2px solid ${colors.white}` : ""};
+        `}
+
   svg:nth-of-type(2) {
     position: absolute;
     right: 1rem;
   }
 `;
 
-interface Props {
+interface Props extends Pick<UploadProps, "theme" | "disabled"> {
   fileName: string;
   fileSize: number;
   base64Content?: string;
   showSize: boolean;
   color: "grey" | "white";
+  errorMessage: unknown;
   isPdfOnly?: boolean;
   handleRemoveFile?: () => void;
+  handleChooseFile?: () => void;
+  style?: React.CSSProperties;
 }
+
 export default function UploadedFile({
   fileName,
   fileSize,
   color,
   base64Content,
   showSize,
   isPdfOnly,
-  handleRemoveFile
+  handleRemoveFile,
+  handleChooseFile,
+  theme,
+  disabled,
+  style,
+  errorMessage
 }: Props) {
   const FileContent = useCallback(() => {
     return (
-      <>
-        {isPdfOnly ? <FilePdf size={23} /> : <ImageSquare size={23} />}
-        <Typography fontSize="1rem" fontWeight="400">
-          &nbsp;&nbsp; {fileName}
-        </Typography>
+      <FileUploadWrapper
+        $isPdfOnly={isPdfOnly}
+        data-disabled={disabled}
+        onClick={() => {
+          if (!isPdfOnly) {
+            handleChooseFile?.();
+          }
+        }}
+        $error={errorMessage}
+        style={{ ...style, ...theme?.overrides }}
+        theme={theme?.triggerTheme}
+      >
+        <Grid flexDirection="row" alignItems="start">
+          {isPdfOnly ? <FilePdf size={23} /> : <ImageSquare size={23} />}
+          <Typography fontSize="1rem" fontWeight="400">
+            &nbsp;&nbsp; {fileName}
+          </Typography>
+        </Grid>
         {showSize && (
           <Typography tag="p">
             <small>{bytesToSize(fileSize)}</small>
           </Typography>
         )}
-      </>
+      </FileUploadWrapper>
     );
-  }, [fileName, isPdfOnly, fileSize, showSize]);
+  }, [
+    fileName,
+    isPdfOnly,
+    disabled,
+    fileSize,
+    showSize,
+    handleChooseFile,
+    theme,
+    style,
+    errorMessage
+  ]);
+
   return (
     <Grid>
-      <AttachmentContainer $isLeftAligned={color === "white"}>
+      <AttachmentContainer
+        $isPdfFile={isPdfOnly}
+        $isLeftAligned={color === "white"}
```

Comment:
I see it was already like this but setting `$isLeftAligned` to `color === "white"` is weird, could you improve this if it's possible?